core/unix/tls_linux_riscv64.c

```diff
@@ -48,29 +48,60 @@
 byte **
 get_dr_tls_base_addr(void)
 {
```

Comment:
This entire file is word-for-word identical to tls_linux_aarchxx.c.  Let's start a convention for sharing among AArchXX and RISCV: there may well be other things that are shared.  Using the term "RISC" comes to mind; hopefully not too close to "RISCV".  So let's rename the existing `tls_linux_aarchxx.c` to `tls_linux_risc.c` and use it on all 3 arches, is my proposal.

Reply:
Great advice, done.